diff --git a/package/other/helloworld/luci-app-ssr-plus/root/usr/bin/ssr-rules b/luci-app-ssr-plus/root/usr/bin/ssr-rules
index 4a85173..f1ead5e 100755
--- a/package/other/helloworld/luci-app-ssr-plus/root/usr/bin/ssr-rules
+++ b/package/other/helloworld/luci-app-ssr-plus/root/usr/bin/ssr-rules
@@ -51,7 +51,8 @@ usage() {
 
 loger() {
 	# 1.alert 2.crit 3.err 4.warn 5.notice 6.info 7.debug
-	logger -st ssr-rules[$$] -p$1 $2
+	logger -st "ssr-rules[$$]" -p"$1" "$2"
+	echo "$(date "+%Y-%m-%d %H:%M:%S"): ssr-rules[$$] $2" >> /var/log/ssrplus.log
 }
 
 flush_r() {
@@ -161,7 +162,7 @@ fw_rule() {
 	$IPT -A SS_SPEC_WAN_FW -d 224.0.0.0/4 -j RETURN
 	$IPT -A SS_SPEC_WAN_FW -d 240.0.0.0/4 -j RETURN
 	$IPT -A SS_SPEC_WAN_FW -p tcp $PROXY_PORTS -j REDIRECT --to-ports $local_port 2>/dev/null || {
-		loger 3 "Can't redirect, please check the iptables."
+		loger 3 "Can't redirect $PROXY_PORTS to $local_port, please check the iptables."
 		exit 1
 	}
 	return $?
