--- a/feeds/packages/utils/antfs-mount/Makefile
+++ b/feeds/packages/utils/antfs-mount/Makefile
@@ -1,7 +1,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=antfs-mount
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -22,7 +22,7 @@ endef
 
 define Package/a/feeds/packages/utils/install
 	$(INSTALL_DIR) $(1)/sbin
-	$(INSTALL_BIN) ./files/mount.ntfs $(1)/sbin
+	$(INSTALL_BIN) ./files/mount.antfs $(1)/sbin
 endef
 
 $(eval $(call BuildPackage,antfs-mount))

--- a/feeds/packages/utils/antfs-mount/files/mount.antfs
+++ b/feeds/packages/utils/antfs-mount/files/mount.antfs
@@ -0,0 +1,3 @@
+#!/bin/sh
+mount -t antfs $@ -o rw
+

--- a/feeds/packages/utils/antfs-mount/files/mount.ntfs
+++ b/feeds/packages/utils/antfs-mount/files/mount.ntfs
@@ -1,2 +0,0 @@
-#!/bin/sh
-mount -t antfs "$@"
